dsl/deploy_apps.rb

```diff
@@ -0,0 +1,89 @@
+require 'rubygems'
+require 'ruby-jmeter'
+require 'cgi'
+require 'net/ssh'
+require './ssh_helper'
+require 'fileutils'
+
+if ARGV.size != 4
+  puts '\n\n------- Tsuru perf test: ERROR! --------'
+  puts '      Usage:'
+  puts '      Please specify a tsuru target domain, a thread'
+  puts '  count and a loop count for each thread.\n '
```

Comment:
Is this usage message and the command in the README correct? It doesn't seem to match the variable assignments of `environment` and `host_suffix` on lines 18-21.